src/pages/about.tsx

```diff
@@ -8,7 +8,7 @@ const About: React.FC = () => {
   return (
     <Layout>
       <Grid sx={{ width: '100%' }}>
-        <ContainerWithBackground element='element2' paddingtop='190px' positionx='left' positiony='32' size='298'>
+        <ContainerWithBackground element='element2' paddingtop='50px' positionx='left' positiony='30' size='298' marginbottom='-170px'>
```

Comment:
It's probably not safe to use CSS position if it's not absolutely necessary. If you'd like, we can go through the problem you're trying to fix together in a shared session?